js/layout-javascript/news-feed-code.js

```diff
@@ -1739,16 +1777,44 @@ DynamicList.prototype.addFilters = function(records) {
 
     var filtersData = {
       filtersInOverlay: _this.data.filtersInOverlay,
+      detailViewOptions: _this.data.detailViewOptions,
       filters: filters
     };
+
     var template = _this.data.advancedSettings && _this.data.advancedSettings.filterHTML
       ? Handlebars.compile(_this.data.advancedSettings.filterHTML)
       : Handlebars.compile(filtersTemplate());
 
     _.remove(filters, function(filter) {
       return _.isEmpty(filter.data);
     });
+
     _this.$container.find('.filter-holder').html(template(filtersData));
+    _this.datePickerFrom = _this.$container.find('#dateFrom');
```

Comment:
jQuery objects should use a `$` prefix.

```suggestion
    _this.$datePickerFrom = _this.$container.find('#dateFrom');
```

Reply:
Done. Thanks